widgets/GCSArgumentSetter-action.json

```diff
@@ -82,12 +82,12 @@
     {
       "name": "ServiceAuthenticationTypeJSON",
       "condition": {
-        "expression": "serviceAccountType == 'JSON'"
+        "expression": "serviceAccountType == 'json'"
```

Comment:
GCSArgumentSetter already had this feature and was merged to develop, since those code changes will be used by all other GC plugins we had to unify GCSArgumentSetter.

Reply:
But why are we changing service account type from json -> JSON?